src/SiteMaster/Core/Auditor/Scan/Progress.php

```diff
@@ -2,7 +2,7 @@
 namespace SiteMaster\Core\Auditor\Scan;
 
 use SiteMaster\Core\Auditor\Site\Page;
-use SiteMaster\Core\Auditor\Site\Pages\Queued;
+use SiteMaster\Core\Auditor\scans\Queued;
```

Comment:
Fixed in https://github.com/mfairchild365/site_master/commit/07d025388010b3784cc0d542aace8b8fa0d699d2